src/app/components/header/header.module.scss

```
@@ -0,0 +1,264 @@
+.header {
+  width: 100%;
+  max-width: 1920px;
+  margin: 0 auto;
+  padding: 10px 0;
+  display: flex;
+  flex-direction: column;
+
+  @media (max-width: 1200px) {
+    max-width: 1200px;
+  }
+
+  @media (max-width: 992px) {
+    max-width: 992px;
+  }
+
+  @media (max-width: 768px) {
+    max-width: 768px;
+  }
+
+  @media (max-width: 576px) {
+    max-width: 576px;
+  }
+}
+
+
+.headerTop {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #ffffff;
+  border-bottom: 1px solid #3f4e65;
+  padding: 5px 5px;
+
+  @media (max-width: 1200px) {
+    padding: 5px 10px;
+  }
+
+  @media (max-width: 992px) {
+    padding: 5px 15px;
+  }
+
+  @media (max-width: 768px) {
+    padding: 5px 20px;
+  }
+
+  @media (max-width: 576px) {
+    padding: 5px 25px;
+  }
+}
+
+
+.header_row {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  @media (max-width: 576px) {
+    flex-direction: column;
+  }
+}
+
+
+.header_left_part {
+  display: flex;
```

```suggestion
}

.header_row {
  display: flex;
  align-items: center;
  width: 100%;

  @media (max-width: 576px) {
    flex-direction: column;
  }
}

.header_left_part {
  display: flex;
```

Между блоками кода должна быть одна пустая линия.